### PR TITLE
Clarify installation instructions

### LINKED ISSUE #1120: Clarify the installation instructions
The default installation instructions should appear clearly on the website:
- install miniconda/anaconda
- `conda install numpy vispy pyqt`
- install the latest proprietary GPU drivers
- run some of the examples
- note that the test suite might be unstable on some systems, which is somewhat unrelated to the working state of the examples

### PR DESCRIPTION
**Pull request by djhoese**
Replaces #2 which fixes https://github.com/vispy/vispy/issues/1120

#2 had a typo and some styling issues that I wanted to fix. The original author hadn't fixed the typo so I copied their PR branch and fixed the issues. I also updated the links to what seem to be the current URLs for the resources referenced in the installation instructions.

### CODE REVIEW

File: installation.rst [hunk 1]
```diff
@@ -45,6 +45,26 @@ your system and video driver. The OpenGL version must be at least 2.1.
...
+`Anaconda <https://www.anaconda.com/download/>`_ will
+install most of the VisPy dependencies for you. If your computer is low on hard
+disk space, or you would like a minimal python installation, you may install
+the `Miniconda <https://conda.io/miniconda.html>`_ package also from
+Continuum Analytics. Once Anaconda is installed, create a
+`conda python environment <https://conda.io/docs/user-guide/tasks/manage-python.html>`_.
+
+Next, install the following VisPy dependencies directly through `pip` or the Anaconda package installer.
+
+$ conda install numpy pyqt
```
Comment (larsoner) [inline installation.rst]: this can be:
```
.. code-block:: console

    $ conda install numpy pyqt
```
and it will render nicely

Comment (larsoner) [reply installation.rst]: (same with the `$ pip install` line below and possibly others)

**Review by larsoner — COMMENTED** [review]
LGTM with or without my aesthetic suggestion

File: installation.rst (hunk 1 above)
Comment (djhoese) [reply installation.rst]: With or without the `$`?

Comment (larsoner) [reply installation.rst]: With, it renders it with highlighting like this

https://6100-1301584-gh.circle-artifacts.com/0/tmp/circle-artifacts.2wP5EEF/html/install_mne_python.html

File: installation.rst [hunk 2]
```diff
@@ -67,8 +95,12 @@ Testing installation
 --------------------
 
 It is strongly advised to run the vispy test suite right after installation to
-check if everything is ok. To do this, just type::
+check if everything is ok. To do this, just type:
+
+.. code-block:: python
```
Comment (larsoner) [inline installation.rst:100]: FYI this one should be unnecessary because the default for the repo should be Python anyway. But no harm in having it

Comment (djhoese) [reply installation.rst:100]: Yeah the code at the top uses the code block directive for python so was trying to be consistent.

Comment (larsoner) [reply installation.rst:100]: I guess it makes the GitHub editor render it nicely, too:

https://github.com/davidh-ssec/vispy-website/blob/05614fa928b7351df32ea7df21eda5870b6a4372/installation.rst

**Pull request merged**